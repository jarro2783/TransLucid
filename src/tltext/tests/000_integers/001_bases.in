%%
0A10;;
0GA;;
0GF;;
0GFF;;
~0GFF;;
0ZF;;
0ZG;;
0ZH;;
0ZY;;
0aZ;;
0zax;;
0xdh;;
