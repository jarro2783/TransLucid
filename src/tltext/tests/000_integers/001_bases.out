10
10
15
255
~255
15
16
17
34
35
2255
2344
